pyanaconda/ui/tui/spokes/time_spoke.py

```diff
@@ -63,8 +142,108 @@ def status(self):
         else:
             return _("Timezone is not set.")
 
+    def _summary_text(self):
+        """Return summary of current timezone & NTP configuration.
+
+        :returns: current status
+        :rtype: str
+        """
+        msg = ""
+        # timezone
+        timezone_msg = _("not set")
+        if self.data.timezone.timezone:
+            timezone_msg = self.data.timezone.timezone
+
+        msg += _("Timezone: %s\n") % timezone_msg
+
+        # newline section separator
+        msg += "\n"
+
+        # NTP
+        msg += _("NTP servers:")
+        if self._ntp_servers:
+            for status in format_ntp_status_list(self._ntp_servers):
+                msg += "\n%s" % status
+        else:
+            msg += _("not configured")
+
+        return msg
 
-    def refresh(self, args = None):
+    def refresh(self, args=None):
+        NormalTUISpoke.refresh(self, args)
+
+        summary = self._summary_text()
+        self._window += [TextWidget(summary), ""]
+
+        if self.data.timezone.timezone:
+            timezone_option = _("Change timezone")
+        else:
+            timezone_option = _("Set timezone")
+
+        _options = [timezone_option, _("Configure NTP servers")]
+        text = [TextWidget(m) for m in _options]
+
+        def _prep(i, w):
```

Comment:
_prep shows up in a number of places. Seems like a good candidate for a helper function, especially since it doesn't depend on anything from the class itself.